backend/msgpack/writer.go

```diff
@@ -97,10 +91,6 @@ func (w *writer) Write(
 	mu unaggregated.MetricUnion,
 	pl policy.PoliciesList,
 ) error {
-	if w.rand.Float64() < w.samplingRate {
```

Comment:
We don't want to record these metrics anymore?

Reply:
Not for now, the ID histogram graph is already broken, and I find these two histograms not that useful. Plus, the random source is not thread-safe. All in all feels like we can just remove these two and add them back when we need them.